wazo_calld/plugins/calls/services.py

```diff
@@ -492,6 +498,37 @@ def set_answered_time(self, channel_id):
             logger.debug('channel %s not found', channel_id)
             raise NoSuchCall(channel_id)
 
+    @staticmethod
```

Comment:
Since this function needs `ari`, what's the advantage to making it a `staticmethod`, instead of using `self._ari`?

Reply:
That's because it is called through `make_call_from_channel` (`calls/services.py:294`) which is also a `staticmethod`. I cannot change it easily without touching multiple other plugins so I avoided changing it to a standard method.

Reply:
Oh yes, of course :) Thanks for the explanation.